From 488a4f13e3c7f4fe0bc9d466b1994424c228afce Mon Sep 17 00:00:00 2001
From: Luna! <51751618+janLuna@users.noreply.github.com>
Date: Sun, 31 Jul 2022 01:30:45 +0200
Subject: [PATCH] Change trans flag colors

Changed trans flag colors to the exact colors of the flag!!!!
--- a/main.c
+++ b/main.c
@@ -73,8 +73,8 @@ const unsigned char codes_gfl[] = {219, 219, 255, 255, 128, 128, 234, 234, 20, 2
 #define FLAG_3_2 (1.334f * M_PI)
 #define FLAG_3_3 (2.0f * M_PI)
 
-#define TRA_BLU (0xa0e0ff)
-#define TRA_PNK (0xffa0e0)
+#define TRA_BLU (0x55cdfc)
+#define TRA_PNK (0xf7a8b8)
 #define TRA_WHT (0xffffff)
 #define TRA_FCT (4.0f)
 
@@ -611,4 +611,4 @@ int main(int argc, char** argv)
             }
         }
     }
-}
\ No newline at end of file
+}
